build/templates/_converters.py.mako

```diff
@@ -222,6 +222,36 @@ def convert_init_with_options_dictionary(values, encoding):
     return init_with_options_string
 
 
+% if config['module_name'] == 'nitclk':
```

Comment:
Similar to Attribute classes, not all drivers use all converters yet the code is there as dead code. I'd leave it alone here as well. Less conditional logic all over the place.

Reply:
See #1039 for a possible solution, but for this PR, always add for all drivers.